Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -368,49 +367,49 @@ function MetalSmelter_JobWorked(j)
 end
 
 function PowerCellPress_UpdateAction(furniture, deltaTime)
-    local spawnSpot = furniture.GetSpawnSpotTile()
-
-    if(spawnSpot.Inventory == nil) then
-        if(furniture.JobCount() == 0) then
-            local itemsDesired = {Inventory.__new("Steel Plate", 10, 0)}
-            local jobSpot = furniture.GetJobSpotTile()
-
-            local j = Job.__new(
-                jobSpot,
-                nil,
-                nil,
-                1,
-                itemsDesired,
-                Job.JobPriority.Medium,
-                false
-            )
-
-            j.RegisterJobCompletedCallback("PowerCellPress_JobComplete")
-            j.JobDescription = "job_power_cell_fulling_desc"
-            furniture.AddJob(j)
-        end
-    else
-        furniture.Parameters["presstime"].ChangeFloatValue(deltaTime)
-
-        if(furniture.Parameters["presstime"].ToFloat() >= furniture.Parameters["presstime_required"].ToFloat()) then
-            furniture.Parameters["presstime"].SetValue(0)
-            local outputSpot = World.current.GetTileAt(spawnSpot.X+2, spawnSpot.y)
-
-            if(outputSpot.Inventory == nil) then
-                World.current.inventoryManager.PlaceInventory( outputSpot, Inventory.__new("Power Cell", 5, 1) )
-                spawnSpot.Inventory.stackSize = spawnSpot.Inventory.stackSize-10
-            else
-                if(outputSpot.Inventory.stackSize <= 4) then
-                    outputSpot.Inventory.stackSize = outputSpot.Inventory.stackSize+1
-                    spawnSpot.Inventory.stackSize = spawnSpot.Inventory.stackSize-10
-                end
-            end
-
-            if(spawnSpot.Inventory.stackSize <= 0) then
-                spawnSpot.Inventory = nil
-            end
-        end
-    end
+	spawnSpot = furniture.GetSpawnSpotTile()
+	
+	if(spawnSpot.Inventory == nil) then
+		if(furniture.JobCount() == 0) then
+			local itemsDesired = {Inventory.__new("plate_steel", 10, 0)}
+			local jobSpot = furniture.GetJobSpotTile()
+
+			local j = Job.__new(
+			jobSpot,
+			nil,
+			nil,
+			1,
+			itemsDesired,
+			Job.JobPriority.Medium,
+			false
+			)
```

Comment:
Maybe also try to indent the parameters by some spaces (except for the closing brace).